src/vs/workbench/contrib/preferences/browser/settingsEditor2.ts

```diff
@@ -387,6 +393,12 @@ export class SettingsEditor2 extends EditorPane {
 		return withUndefinedAsNull(cachedState);
 	}
 
+	override getViewState(): object | undefined {
```

Comment:
How is this different from the other cached state with `ISettingsEditor2State`? Is there overlap or is that a totally different scenario?

Reply:
That scenario is for reloads, though I noticed that these two implementations can interfere with each other sometimes, so I'll have to fix that first.